Packs/FortiSIEM/IncidentFields/incidentfield-FortiSIEM_Incident_Last_Seen.json

```diff
@@ -0,0 +1,29 @@
+{
+    "associatedToAll": false,
+    "associatedTypes": [
+        "FortiSIEM"
+    ],
+    "caseInsensitive": true,
+    "cliName": "incidentlastseen",
```

Comment:
Please add prefix to cliname.
should be `fortisiemincidentlastseen`

Reply:
Done.